opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -77,6 +83,21 @@ export const findAll = async (context, user, args) => {
   return listEntitiesPaginated(context, user, types, args);
 };
 
+export const findAllRestricted = async (context, user, args) => {
```

Comment:
Would be better to add detail on restricted, because some entity can be restricted because of marking or organization. Feel free to use another name than my proposal.
```suggestion
export const findAllAuthMemberRestricted = async (context, user, args) => {
```